Framework/Core/src/CommonServices.cxx

```diff
@@ -154,24 +154,24 @@ o2::framework::ServiceSpec CommonServices::datatakingContextSpec()
     .configure = noConfiguration(),
     .preProcessing = [](ProcessingContext& processingContext, void* service) {
       auto& context = processingContext.services().get<DataTakingContext>();
-      // Only on the first message
-      if (context.source == OrbitResetTimeSource::Data) {
-        return;
-      }
-      // Only if we do not have already the proper number from CTP
-      if (context.source == OrbitResetTimeSource::CTP) {
-        return;
-      }
-      context.source = OrbitResetTimeSource::Data;
-      context.orbitResetTime = -1;
+      //// Only on the first message
+      // if (context.source == OrbitResetTimeSource::Data) {
+      //   return;
+      // }
+      //  Only if we do not have already the proper number from CTP
+      // if (context.source == OrbitResetTimeSource::CTP) {
+      //   return;
+      // }
+      // context.source = OrbitResetTimeSource::Data;
+      // context.orbitResetTime = std::to_string(-1);
       for (auto const& ref : processingContext.inputs()) {
         const o2::framework::DataProcessingHeader *dph = o2::header::get<DataProcessingHeader*>(ref.header);
         const auto* dh = o2::header::get<o2::header::DataHeader*>(ref.header);
         if (!dph || !dh) {
           continue;
         }
-        LOGP(debug, "Orbit reset time from data: {} ", dph->creation);
-        context.orbitResetTime = dph->creation;
+        // LOGP(debug, "Orbit reset time from data: {} ", dph->creation);
+        // context.orbitResetTime = std::to_string(dph->creation);
```

Comment:
also this assignment invalidates the fetching from the server. Actually, I would eliminate all other ways of getting the orbit reset time but reading from the CCDB, for synthetic runs the trick with DPL_RAWPROXY_OVERRIDE_ORBITRESET is fully enough and still leads to reading correct OrbitReset objects from the CCDB. 
In opposite, I realize that hijacking the `orbit-reset-time` at https://github.com/AliceO2Group/AliceO2/pull/9472/files#diff-75f33822fa1864e701f55c79b380b588f9d8d571edf106891acb84a469d4ea7eL209 is prone to create problems: it will provide some orbit reset time generate a timestamp for reading other objects, but many workflows request the orbir_reset object itself, so it should be anyway read. Shall we eliminate this line 209 (it is ok to keep `>GetProperty<std::string>("orbit-reset-time"...` in the proxy) and revert conditional reading of the orbitReset object?

Reply:
Yes, sounds good to me.